data/human/human missions.txt

```diff
@@ -3767,11 +3767,66 @@ mission "Earth Retirement"
 			`	As you count up the large clump of credits you were handed (the sum comes out to the expected <payment>), they slowly make their way to the train station. Presumably, their new apartment is far from here.`
 				accept
 			label reject
+			action
+				event "charles donna timer" 90
 			`	Charles looks surprised for a moment, and then, for the first time you've seen, he smiles. "Thank you, Captain <last>. I wish there were more people like you in the Paradise Worlds."`
 			`	You wish the couple the best of luck on Earth, and they slowly make their way to the train station. Presumably, their new apartment is far from here.`
 
 
 
+event "charles donna timer"
+
+
+
+mission "Earth Retirement 2"
+	name "Retired Family to Shangri-La"
+	description "Transport two seniors and their son to <destination>, where they plan to retire. They will pay you <payment>."
+	minor
+	source "Earth"
+	destination "Shangri-La"
+	to offer
+		has "event: charles donna timer"
+		has "outfit: Luxury Accommodations"
+	passengers 3
+	on offer
+		conversation
+			`As you enter the spaceport, you see a familiar sight. The elderly couple Charles and Donna Singer, who you helped move to Earth a while ago, are talking to a young well-dressed man. Donna looks much the same as you remember her, but Charles is sporting a bandage on his head, and one of his arms looks like it's in a sling. When they see you, Donna points excitedly in your direction, and the entire group moves to meet you.`
+			`	"Captain! I never thought we would see you again," Donna begins. "I'd like to introduce you to my son, Abel." She gestures to the young man, who shakes your hand vigorously. Now that you're close, you can see that he is dressed very finely. His suit is made of expensive wool with ebony buttons, his shoes are emblazoned with designer logos, and you can see a jewel-encrusted watch peeking out from underneath the well-tailored cuff of one arm.`
+			`	"It's great to meet you, Captain...?"`
+			choice
+				`	"<first> <last>."`
+				`	"<last>."`
+			`	"Captain <last>," he smiles. "How very nice to meet you indeed. My parents here told me what you did for them. If I may introduce myself more fully, I am Abel Singer, Junior Vice President of Marketing Operations for Syndicated Systems. I want to thank you for going out of your way to help my parents when they needed it most."`
+			choice
+				`	"It was nothing."`
+				`	"What happened to Charles?"`
+					goto injuries
+				`	"How could you leave them to rot...?"`
+					goto rot
+			`	"It was not nothing," Donna blurts out. "You were there for us when nobody else cared. You didn't even accept the agreed upon payment! Our former employers, our friends, even our children..." She trails off.`
```

Comment:
The player could have taken the payment in the last mission, so I think that line should be removed or you need to add a condition to the last mission & a branch here.

Reply:
This won't be offered if the player took the payment.

Reply:
Ah, I see. My bad, this works as is -- feel free to resolve